InstrumentDB/1 Table.sql

```diff
@@ -0,0 +1,24 @@
+use InstrumentDB
+go
+
+drop table if exists dbo.Instrument
+go
+
```

Comment:
-- LL General Tip: All constraint names should follow the following naming convention: ck_TableName_ColumnName_ConstraintText (you are missing the table name)